src/App.vue

```diff
@@ -85,13 +95,17 @@ import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
 import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
 import SbcFeeSummary from 'sbc-common-components/src/components/SbcFeeSummary.vue'
 import { EntityInfo, Stepper, Actions } from '@/components/common'
+import {
+  AddPeopleAndRoles, CreateShareStructure, DefineCompany, ReviewConfirm,
+  SignIn, SignOut
+} from '@/views'
```

Comment:
Is it possible to `import * from ...` for future support?

Reply:
Updated.